Sources/Caching/DeviceCache.swift

```diff
@@ -142,25 +139,48 @@ class DeviceCache {
         }
     }
 
-    // MARK: - offerings
+    // MARK: - Offerings
 
-    func cache(offerings: Offerings) {
-        offeringsCachedObject.cache(instance: offerings)
+    func cachedOfferingsData(appUserID: String) -> Data? {
+        return self.userDefaults.read {
+            $0.data(forKey: CacheKey.offerings(appUserID))
+        }
+    }
+
+    func cache(offerings: Offerings, appUserID: String) {
+        self.cacheInMemory(offerings: offerings)
+
+        if let jsonData = try? JSONEncoder.default.encode(value: offerings.response, logErrors: true) {
+            self.userDefaults.write {
+                $0.set(jsonData, forKey: CacheKey.offerings(appUserID))
+            }
```

Comment:
I mentioned it in Linear, not a fan of duplicating the information. We should be careful that the memory and the disk caches don't become out of sync...

But probably not a reason to block the PR either.

Reply:
Ahh yeah makes sense, I missed that. We will have the same situation in Android actually. So yeah I think this approach makes sense.

One thing we could do to improve this is have an "offerings cache" and a "offerings http response cache" to try to differentiate them more?